src/main/java/com/solvd/carsharing/web/controller/CarController.java

```diff
@@ -0,0 +1,57 @@
+package com.solvd.carsharing.web.controller;
+
+import com.solvd.carsharing.aggregate.CarAggregate;
+import com.solvd.carsharing.command.CommandService;
+import com.solvd.carsharing.domain.Car;
+import com.solvd.carsharing.event.Event;
+import com.solvd.carsharing.query.FindAllCarsQuery;
+import com.solvd.carsharing.query.FindAllEventsByCarNumberQuery;
+import com.solvd.carsharing.query.QueryService;
+import com.solvd.carsharing.web.dto.CarAggregateDto;
+import com.solvd.carsharing.web.dto.CarDto;
+import com.solvd.carsharing.web.dto.EventDto;
+import com.solvd.carsharing.web.mapper.CarMapper;
+import com.solvd.carsharing.web.mapper.EventMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
```

Comment:
@vpetrovaa, don't use wildcard import, in this code I can't see which classes are imported and so on..